applications/peripheral_sensor_node/configuration/nrf54l15pdk_nrf54l15_cpuapp/app.overlay

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/{
+	/delete-node/ leds;
+
+	leds0 {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio2 9 GPIO_ACTIVE_HIGH>;
```

Comment:
On pca63565 GPIO2.09 is used for SPI_MISO signal
https://github.com/nrfconnect/sdk-nrf/blob/main/boards/shields/pca63565/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay#L40

Reply:
Good point, changed to LED1.